src/mscorlib/src/Microsoft/Win32/Win32Native.cs

```diff
@@ -916,12 +916,12 @@ internal static bool DoesWin32MethodExist(String moduleName, String methodName)
         // a web client asking a server for "http://server/com1.aspx" and
         // then causing a worker process to hang.
         [System.Security.SecurityCritical]  // auto-generated
-        internal static SafeFileHandle SafeCreateFile(String lpFileName,
+        internal static Win32NativeCloseHandle SafeCreateFile(String lpFileName,
```

Comment:
It finally dawned on me that all I need to do is make CreateFile conditional on CoreCLR and Unix. That is is the only place we create the SafeFileHandle- in the Unix/CoreCLR case I'll get the handle from the CoreFX code. I believe that should be good enough.